packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -29,26 +34,51 @@ export const styles = (theme) => ({
     ...theme.typography.body2,
     minHeight: 'auto',
   },
+  indicatorWrapper: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  indicator: {
+    marginLeft: theme.spacing(2)
+  }
 });
 
 const MenuItem = React.forwardRef(function MenuItem(props, ref) {
   const {
+    children: childrenProp,
     classes,
     className,
     component = 'li',
     disableGutters = false,
+    nestedItems,
+    NestedMenuIndicator = KeyboardArrowRight,
+    NestedMenuProps = {},
+    openNestedMenu = false,
+    onKeyDown,
     role = 'menuitem',
     selected,
+    handleArrowRightKeydown,
+    setParentLastEnteredItemIndex,
     tabIndex: tabIndexProp,
     ...other
   } = props;
 
+  const listItemRef = React.useRef(null);
+  const handleOwnRef = React.useCallback((instance) => {
+    // #StrictMode ready
```

Comment:
whats this do?

Reply:
I'm not sure. It comes from the `upstream/master` merge. I think it's some special comment that informs their testing and verification process?

Reply:
We are following the MUI pattern here: https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Menu/Menu.js#L121